ios/BT/ExposureManager.swift

```diff
@@ -104,24 +104,35 @@ final class ExposureManager: NSObject {
 
   // MARK: == State ==
 
-  enum EnabledState: String {
-    case enabled = "ENABLED"
+  enum ExposureNoticationStatus: String {
+    case unknown = "UNKNOWN"
```

Comment:
What do we think about downcasing all of these? so `Unknown`, to follow the convention we have in the JS layer.